cmd/contour/contour.go

```diff
@@ -59,10 +56,14 @@ func main() {
 	path := bootstrap.Arg("path", "Configuration file.").Required().String()
 	bootstrap.Flag("admin-address", "Envoy admin interface address").StringVar(&config.AdminAddress)
 	bootstrap.Flag("admin-port", "Envoy admin interface port").IntVar(&config.AdminPort)
+	bootstrap.Flag("xds-address", "xDS gRPC API address").StringVar(&config.XDSAddress)
+	bootstrap.Flag("xds-port", "xDS gRPC API port").IntVar(&config.XDSGRPCPort)
 
 	serve := app.Command("serve", "Serve xDS API traffic")
 	inCluster := serve.Flag("incluster", "use in cluster configuration.").Bool()
 	kubeconfig := serve.Flag("kubeconfig", "path to kubeconfig (if not in running inside a cluster)").Default(filepath.Join(os.Getenv("HOME"), ".kube", "config")).String()
+	serve.Flag("xds-address", "xDS gRPC API address").Default("127.0.0.1").StringVar(&config.XDSAddress)
```

Comment:
please capture these values directly, rather than stringvar, ie
```
xdrAddress := serve.Flag("xds-address", "xDS gRPC API address").Default("127.0.0.1").StringVar()
```